src/scraper.py

```diff
@@ -87,53 +84,38 @@ def get_points_change(self):
         self.points_percent['points'] = points
         self.points_percent['percent'] = percent
 
-        # Store dict in all_data_dict
-        self.dict['points_change'] = self.points_percent
-
         return self.points_percent
 
 
     def get_current(self):
         '''Parse self.parse_points_close (this is a list, current price is index 0) for previous close, return self.current'''
 
         self.current = (self.parse_points_close.contents[0]).string
-        self.dict['current'] = self.current
         return self.current
 
 
     def get_cap(self):
         '''Parse self.parse_cap for market cap, return self.cap'''
 
         self.cap = (self.parse_cap.contents[0]).string
-        self.dict['market_cap'] = self.cap
         return self.cap
 
     
     def get_volume(self):
         '''Parse self.parse_volume for volume, return self.volume'''
 
         self.volume = (self.parse_volume.find('span')).string
-        self.dict['volume'] = self.volume
         return self.volume
 
     
     def get_avg_volume(self):
         '''Parse self.parse_avg_volume for average volume, return self.avg_volume'''
 
         self.avg_volume = (self.parse_avg_volume.find('span')).string
-        self.dict['avg_volume'] = self.avg_volume
+        # self.dict['avg_volume'] = self.avg_volume
```

Comment:
Any reason why this line has been commented out and not deleted?

Reply:
Now fixed in [this commit](https://github.com/BoringStocks/StonkScraper/pull/9/commits/1f82422ee1c49c1cd4a364028882ab38111bb49a)